internal/node/node.go

```diff
@@ -133,6 +136,7 @@ func (n *KoinosP2PNode) handleTransactionBroadcast(topic string, data []byte) {
 	}
 	binary := types.NewVariableBlob()
 	binary = trxBroadcast.Serialize(binary)
+	log.Infof("Publishing Transaction: %s", util.TransactionString(&trxBroadcast.Transaction))
```

Comment:
Can we make this `Publishing transaction: ` to be consistent with other microservice logs?